cypress/integration/pages/index.spec.ts

```diff
@@ -20,4 +20,24 @@ describe('Homepage', () => {
 
 		cy.get('h1').should('be.visible').should('contain', 'How much am I');
 	});
+
+	it('footer is visible', () => {
+		cy.visit('/');
+
+		cy.get('footer')
+			.should('be.visible')
+			.find('a[href="/faq"]')
```

Comment:
```suggestion
			.contains('F.A.Q.')
```

I think it would suit more here, wdyt? https://docs.cypress.io/guides/references/best-practices.html#Text-Content

Reply:
Ah beautiful. I was trying to do something like that but didn't see it in the docs in my short trip. It felt a little silly doing it my way. "Look for X, then check for the same X" was iffy.